src/components/NavBar /ShoppingCartItem.jsx

```diff
@@ -0,0 +1,63 @@
+import { Box, Button, Text } from "@chakra-ui/react";
+import React, { useState } from "react";
+import SectionSeperator from "./SectionSeperator";
+
+const ShoppingCartItem = ({
+  planName,
+  imageNum,
+  commentNum,
+  shoppingCartItems,
+  setShoppingCartItems,
+}) => {
+  const [hasCartItem, setHasCartItem] = useState(true);
+
+  const toggleRemovebtn = () => {
+    if (shoppingCartItems) {
+      setShoppingCartItems(shoppingCartItems.slice(1));
+    }
+    setHasCartItem(false);
+  };
+
+  const imageURL =
```

Comment:
for now download the image and place it into the `/public` folder and import it from there when you want to render this image.

Reply:
basically don't use the actual url from the github because after I delete this issue, the image wont show lol